windows.fnl

```diff
@@ -490,8 +479,7 @@
   (let [screens (hs.screen.allScreens)]
```

Comment:
Please update the comment above

Reply:
ok. Would you mind also looking at https://github.com/agzam/spacehammer/pull/203 pls?

Reply:
I mean the comment for `enter-window-menu`

Reply:
ok, got it. Thank you! Good catch; it's updated now.